### ISSUE TITLE
Block textures all black

### ISSUE THREAD

**Issue by gigaprofisi** [PeytonPlayz595/Beta-1.7.3#11]
All block textures load improperly. Entity textures, particles, interfaces, and all else still appear fine

![image](https://github.com/user-attachments/assets/063bf83e-4074-4260-ba78-161d32f6cfac)

**Comment by PeytonPlayz595** [comment]
Yes, this is a bug caused by texture mipmapping, I don't have the time to work on this project at the moment but a temporary fix is to turn anaglyph on and off in settings

**Comment by chicken-nugget1104** [comment]
i just made a PR to fix this issue #16 

this would be a more permanent fix, as it runs on tickrun, and only occurs once to minimize lag.
